src/components/TimePicker/TimePicker.tsx

```diff
@@ -452,58 +455,52 @@ function TimePicker({forwardedRef, defaultValue, onSubmit, onInputChange}) {
                 >
                     <AmountTextInput
                         placeholder={numberFormat(0)}
-                        maxLength={2}
                         formattedAmount={hours}
                         onKeyPress={(e) => {
                             lastPressedKey.current = e.nativeEvent.key;
                         }}
                         onChangeAmount={handleHourChange}
-                        role={CONST.ACCESSIBILITY_ROLE.TEXT}
                         ref={(ref) => {
                             if (typeof forwardedRef === 'function') {
-                                forwardedRef({refHour: ref, minuteRef: minuteInputRef.current});
-                            } else if (forwardedRef && _.has(forwardedRef, 'current')) {
+                                forwardedRef({refHour: ref as TextInput | null, minuteRef: minuteInputRef.current});
+                            } else if (forwardedRef && 'current' in forwardedRef) {
                                 // eslint-disable-next-line no-param-reassign
-                                forwardedRef.current = {hourRef: ref, minuteRef: minuteInputRef.current};
+                                forwardedRef.current = {refHour: ref as TextInput | null, minuteRef: minuteInputRef.current};
```

Comment:
This is a bit unconventional, `refHour` vs `minuteRef` - how about renaming so we end up with `hourRef` and `minuteRef`